Bottom: 274477a847124e862d089b22912bb70d7eb669f3
Top:    9064d7b536756156292698598ee4887c02af824d
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 09:53:47 -0500

Refresh of TESTING--VirtualMachine

---
diff --git a/vmdaedalus/vagrant_ansible_inventory b/ansible-playbook/vmdaedalus
similarity index 55%
rename from vmdaedalus/vagrant_ansible_inventory
rename to ansible-playbook/vmdaedalus
index 51db39f0..dc2fd1e9 100644
--- a/vmdaedalus/vagrant_ansible_inventory
+++ b/ansible-playbook/vmdaedalus
@@ -1,7 +1,5 @@
-# Generated by Vagrant
-
 vmdaedalus.novatech-llc.com ansible_host=127.0.0.1 ansible_port=10022 ansible_user='vagrant' ansible_ssh_private_key_file='/home/coopera/Developer/testdaedalus/.vagrant/machines/vmdaedalus/virtualbox/private_key'
-bbworker ansible_host=127.0.0.1 ansible_port=20022 ansible_user='vagrant' ansible_ssh_private_key_file='/home/coopera/Developer/testdaedalus/.vagrant/machines/bbworker/virtualbox/private_key'
+buildbot-worker.novatech-llc.com ansible_host=127.0.0.1 ansible_port=20022 ansible_user='vagrant' ansible_ssh_private_key_file='/home/coopera/Developer/testdaedalus/.vagrant/machines/bbworker/virtualbox/private_key'
 
 [bacula]
 vmdaedalus.novatech-llc.com
@@ -11,7 +9,7 @@ vmdaedalus.novatech-llc.com
 
 [docker-hosts]
 vmdaedalus.novatech-llc.com
-bbworker
+buildbot-worker.novatech-llc.com
 
 [buildbot-worker]
-bbworker
+buildbot-worker.novatech-llc.com
